### ISSUE TITLE
Review Cable Area Symbology

### ISSUE THREAD

**Issue by TomRichardson6** [iho-ohi/S-101_Portrayal-Catalogue#452]
Paper ENCWG9-03.04.1 includes an observation that Submarine Cable Areas with category of ferry display with symbols for power cables. 

Issue 59 corrected this issue for Submarine Cable features but action may be needed to ensure Cable Areas symbology is consistent.

https://github.com/iho-ohi/S-101_Portrayal-Catalogue/commit/985a016c621c142b325d4d1f9f56f6ff18d3867f

This issue proposes that the consistency between the two rules is reviewed.

**Comment by DavidGrant-NIWC** [comment]
![image](https://github.com/user-attachments/assets/0828b927-40e5-4bae-93e7-de017d5bbaa7)

What is the proper symbol?
* S-101 uses the same symbology as S-52: S-101 `CableArea` portrayal is based on S-52 `CBLARE`
  * S-52 `CATCBL` doesn't provide ferry as an option, so there is no direct S-52 equivalent for the S-101 encoding.

Recommend that the PSWG provide an updated portrayal / symbol for this case.
* In particular, a centered area symbol is needed.
* The correction made in issue #59 did not require a symbol; it depicts a curve using a thick magenta dashed line.
* I assume the area outline for `CableArea` with `categoryOfCable`=`7 - Ferry` should be updated to use a thick dashed magenta line for both plain and symbolized boundary options.
  * Should the line style change be made independently of PSWG input?
    * If so, when should the line style change be made?
* What is the target PC version for these changes?

**Comment by alvarosanuy** [comment]
I tried to map possible options moving forward.
Are they in line with the discussions at ENCWG9 and what S-52 would implement (noting current differences in category of cable attribution options)? 
**Need help with creating any new Point and Complex Line symbols**

![image](https://github.com/user-attachments/assets/ebc1c866-6572-4db5-b202-dc4781f720e6)
![image](https://github.com/user-attachments/assets/ce69347f-5235-432a-8ec1-03ecfcfbffb6)

**Comment by DavidGrant-NIWC** [comment]
> Need help with creating any new Point and Complex Line symbols

We could convert whatever symbols the ENCWG develops. They probably won't develop a symbol for our "?" centered area symbol though since I don't think they have the ability to encode that feature/attribute/geometry combination.

**Comment by MikusRL** [comment]
It was my understanding at the ENCWG9 meeting that for the S-57 ENCs only the telecomunication category was requested to become distinguishable, and that the "flash" symbol remains as the "default" for all other categories. So for the individual cables the lines would always show the flash, except when the telephone or the telegraph category is encoded. 

Similarly then with the areas - The flash is a "default" symbol in the border and center, except where the areas are purely of the telephone or telegraph cables. That then would use the new border line AND new center symbol: big light magenta "wigle" element.

You also have missed the agreed new overhead cable symbol. For power line the light gray "flash" symbol would be added to the gray dashed line. For all other categories the "default " stays as the grey dashed line.

**Comment by alvarosanuy** [comment]
Thanks for the update Mikus.
> That then would use the new border line AND new center symbol: big light magenta "wigle" element.

I think we can do better regarding the central symbol here ... Prefer one of the options proposed in my table above.

I recommend that, for S-101, we proceed as recommended in my tables above. having said this, I would like to coordinate the same 'central symbol' for 'Telecommunication Cables' with S-52. Who was tasked to develop work in the new symbology (*DAI) for S-52??

@TomRichardson6 - Your view?

**Comment by DavidGrant-NIWC** [comment]
> Similarly then with the areas - The flash is a "default" symbol in the border and center, except where the areas are purely of the telephone or telegraph cables. That then would use the new border line AND new center symbol: big light magenta "wigle" element.

Ok, but what is the proper symbol for cable areas of type ferry?
![image](https://github.com/user-attachments/assets/7e81f528-df8e-44e8-abdb-1ed713702468)

**Comment by alvarosanuy** [comment]
**DECISIONS & ACTIONS ARISING FROM ENCWG9 2024**

![image](https://github.com/user-attachments/assets/65e74e78-4264-47e8-b7d2-89b3c07721bf)
![image](https://github.com/user-attachments/assets/db937115-5762-4b43-b9d9-9b835bd65d32)
![image](https://github.com/user-attachments/assets/97d02390-d41c-4743-a824-0522dbf77fc3)

**Comment by TomRichardson6** [comment]
![image](https://github.com/user-attachments/assets/ce07da83-b267-4059-9e9e-cabb39a845ca)

Looking at a real example in S-57 we see encoding of both the ferry route area and a Cable Area with an anchoring prohibited restriction. With no value of CATCBL. Noting input from @alvarosanuy perhaps the discussion could be that restricted area should be encoded but this would be a problem for conversion from S-57 to S-101. Overall I think it should be retained but with appropriate symbology. 

To be clear this is going to be a post 2.0.0 change for discussion by the Port SG and at PT14.

@alvarosanuy please can you raise a PortSG issue and this one can then be closed?

**Comment by JeffWootton** [comment]
@alvarosanuy : A cable area is not only used to encode an area within which there are numerous cables. It is also used to encode where "a regulatory authority designates an area for the protection of a cable, or cables." (DCEG clause 14.3.1). This is the reason why `restriction` is an allowable attribute for the feature `CableArea`. I therefore do not have a problem with value _7_ being an allowable value for `categoryOfCable` on `CableArea`, as the cable may have a regulatory surrounding area within which restrictions, such as stopping, anchoring ...) apply.

**Comment by JeffWootton** [comment]
Following on from my last comment, I think we may have overthought the addition of value _7_ for `categoryOfCable` given that the `FerryRoute` feature has attribute ferry value _2_ (cable ferry). I think we have now created 2 ways of encoding the same real-world instance, which should be avoided.

In terms of encoding a ferry cable and an associated area within which restrictions apply, I suggest that this should be done by encoding a FerryRoute of geometric primitive line to represent the cable position; and a `CableAre`a feature having `categoryOfCable` = _7_ (ferry) to encode the area in which restrictions apply. From a modelling perspective, this would mean retaining value _7_ for `categoryofCable` but removing it as an allowable value for the `CableSubmarine` feature. The guidance should then be amended accordingly.

I will open a new GitHub Issue in the Documentation and FC GitHub repository to begin the discussion on this for post-Edition 2.0.0.

**Comment by DavidGrant-NIWC** [comment]
On hold pending results of https://github.com/iho-ohi/S-101-Documentation-and-FC/issues/182 and PSWG guidance

**Comment by alvarosanuy** [comment]
> Looking at a real example in S-57 we see encoding of both the ferry route area and a Cable Area with an anchoring prohibited restriction.

I would encode this case as a **FerryRoute/categoryOfFerry**=2 (cable Ferry) sitting within a **CautionArea** or **RestrictedArea/restriction**=1 (anchoring prohibited), 24 (dragging prohibited)    [for example]

I'm not sure we need a **CableArea** **categoryofCable**=7 in S-101 ...

**Comment by alvarosanuy** [comment]
> A cable area is not only used to encode an area within which there are numerous cables. It is also used to encode where "a regulatory authority designates an area for the protection of a cable, or cables." (DCEG clause 14.3.1)

Agreed but, I believe that, in real life, you won't find an area of 'cable ferry' cables without a working ferry route associated to them.
It may make more sense to add '**restriction**' as a valid attribute to **FerryRoute** and with that, avoid the need of double encoding a **RestrictedArea** on top. 
**If** the ferry has been disconnected from the cables and removed for repairs (for example) the encoder could simply add status = 4 (not in use) and leave the **restriction** attribute populated (when/if included in **FerryRoute**) or in the overlapping **RestrictedArea**.
In this example, I wouldn't delete the **FerryRoute** and replace it with a **CableArea/categoryOfCable**=7 (ferry).

**Comment by alvarosanuy** [comment]
**PsWG meeting on 03/12/24**

It was decided to replicate the mapping & symbology developed by the ENCWG.

Discussions on 'Ferry Cables' is to occur in the Documentation GitHub space and any decisions impacting current symbolization referred back to the PsWG.

- [ ] **7Cs** (Olaf/Holger) to upload relevant mapping rules and symbol files (including engineering drawings) developed for S-52 to this issue at the earliest opportunity.
- [ ] **NIWC** to implement corresponding S-101 PC changes in the first post PC 2.0.0 version.
